service_configuration_lib/utils.py

```diff
@@ -28,3 +35,52 @@ def load_spark_srv_conf(preset_values=None) -> Tuple[Mapping, Mapping, Mapping,
     except Exception as e:
         log.warning(f'Failed to load {DEFAULT_SPARK_RUN_CONFIG}: {e}')
         raise e
+
+
+def ephemeral_port_reserve_range(preffered_port_start: int, preferred_port_end: int, ip='127.0.0.1') -> int:
```

Comment:
there's a small typo in `preffered_port_start`: `preferred_port_start` is probably what we wanted :)

Reply:
Fixed. Thanks for point out